docs/features/feature-guides/compliance-forms/create-a-form.md

```diff
@@ -0,0 +1,159 @@
+       alt="Create a new Compliance Form"/>
+</p>
+
+### Step 2: Add Questions to your Form
+
+Next, create Questions you want your users to complete to capture the desired metadata with this Compliance Form. There are five types of Questions that can be created, each of which can be set to be **required to respond**:
+
+* **Ownership:** Assign one or more Owners to the Asset, with the option to predefine the set of allowed Owners and/or Ownership Types.
+    * _E.g. Who is responsible for ensuring the accuracy of this Dataset?_
+* **Domain:** Assign a Domain to the Asset, with the option to predefine the set of allowed Domains.
+    * _E.g. Which Domain does this Dashboard belong to? Sales, Marketing, Finance._
+* **Documentation:** Provide Documentation about the Asset and/or Column.
+    * _E.g. What is the primary use case of this Dataset? What caveats should others be aware of?_
+* **Glossary Terms:** Assign one or more Glossary Term to the Asset and/or Column, with the option to predefine the set of allowed Glossary Terms. 
+    * _E.g. What types of personally identifiable information (PII) are included in this Asset? Email, Address, SSN, etc._
+* **Structured Properties:** Apply custom properties to an Asset and/or Column.
+    * _E.g. What date will this Dataset be deprecated and deleted?_
+
+When creating a Question, be sure to give it an easy-to-understand Title, and provide additional context or direction in the Description.
+
+<p align="center">
+  <img
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-prompt.png"
+       alt="Create a new Compliance Form prompt"/>
+</p>
+
+### Step 3: Assign your Form to relevant Assets
+
+Now that you have defined the Questions you want Users to complete, it's now time to select the in-scope Assets for this exercise.
+
+In the **Assign Assets** section, you can easily target the specific set of Assets that are relevant for this Form with the following steps:
+
+1. Add a Condition or Group of Conditions
+2. Choose the appropriate filter type, such as:
+    * Asset Type (Dataset, Chart, etc.)
+    * Platform (Snowflake, dbt, etc.)
+    * Domain (Sales, Marketing, Finance, etc.)
+    * Assigned Owners
+    * Assigned Glossary Terms
+3. Decide between **All**, **Any**, or **None** of the filters should apply
+4. Preview the relevant Assets to confirm you have applied the appropriate filters
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-assign-assets.png"
+       alt="Assign assets to a Compliance Form"/>
+</p>
+
+### Step 4: Add Recipients to your Form
+
+Now that you have defined the set of Questions to be answered for a set of Assets, it's now time to delegate out to your Users.
```

Comment:
Now that you've defined your a) Questions and b) Selected Assets (or Assigned Assets), we can select the assignees, or the users & groups who will be required to complete the form for each selected asset.

Reply:
I feel like in the product we should change our language from Add Recipients to Add Assignees. Mainly because recipient feels more like I'm receiving a beautiful present and assignee is more realistic - I'm being assigned some work to complete.

Reply:
agreed -- it should be changed to assignees in-product. Will use "Assignees" in the docs for now